README.md

```diff
@@ -1,23 +1,29 @@
-# Implement Anonymous Classes for Greetings
 
-In this task, you're given a base class named `HelloWorldAnonymousClasses` containing an interface `HelloWorldGreeting` with two methods: `greet()` and `greetSomeone(someone: String)`. 
-Your objective is to implement the `sayHello(names: List<String>): List<String>` method using anonymous classes to provide greetings in English, French, and Spanish.
+# Implementation of Anonymous Classes for Greetings
 
-- Start by creating three anonymous classes within the `sayHello` method, each implementing the `HelloWorldGreeting` interface. 
-- Each class should provide its own implementation of the `greet()` and `greetSomeone(someone: String)` methods:
-  - An `EnglishGreeting` that greets "world" or a specific person in English.
-  - A `FrenchGreeting` that greets "tout le monde" or a specific person in French.
-  - A `SpanishGreeting` that greets "mundo" or a specific person in Spanish.
+In this task, you're provided with a base class named `HelloWorldAnonymousClasses`. It contains an interface `HelloWorldGreeting` with two methods: `greet()` and `greetSomeone(someone: String)`. 
 
-- Each greeting class should have a `greet()` method that returns a general greeting to the world, and a `greetSomeone(someone: String)` method that returns a personalized greeting to the specified person.
+**Your goal:** implement the `sayHello(names: List<String>): List<String>` method using anonymous classes to provide greetings in **English, French,** and **Spanish**.
 
-- In the `sayHello` method, create a list of these greeting objects and use it to compile a list of general greetings from each language (e.g. "Hello world").
+## Instructions:
 
-- Then, for each name in the `names` parameter, generate personalized greetings in all three languages and append them to the list of general greetings.
+ 1. Begin by creating three anonymous classes within the `sayHello` method, each implementing the `HelloWorldGreeting` interface.
+    
+ 2. Each class should offer its own implementation of the `greet()` and `greetSomeone(someone: String)` methods:
+    -   An `EnglishGreeting` that greets “world” or a specific person in English.
+    -   A `FrenchGreeting` that greets “tout le monde” or a specific person in French.
+    -   A `SpanishGreeting` that greets “mundo” or a specific person in Spanish.
+      
+ 3. Each greeting class should include a `greet()` method that returns a general greeting to the world and a `greetSomeone(someone: String)` method that returns a personalized greeting to the specified person.
+    
+ 4. In the `sayHello` method, create a list of these greeting objects and utilize it to compile a list of general greetings from each language (e.g., “Hello world”).
```

Comment:
4. In the `sayHello` method, create a list of these greeting objects and utilize it to compile a list of general greetings from each language (e.g., `Hello world`).